source/code/plugins/scom_common.rb

```diff
@@ -66,7 +66,13 @@ def self.parse_json_record_encoding(record)
     def self.create_secure_http(uri)
       http = Net::HTTP.new( uri.host, uri.port )
       http.use_ssl = true
-      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+      if SCOM::Configuration.enable_server_auth
+        OMS::Log.info_once("Enabling server certificate validation")
+        http.verify_mode = OpenSSL::SSL::VERIFY_PEER
+      else
+        OMS::Log.info_once("Disabling server certificate validation")
```

Comment:
@lagalbra 
Yes it does log three times. Since an in memory hash is maintained to check whether the message is already logged or not, it gets reset on every OMSAgent restart.